.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+## What this PR does / why we need it:
+
+## Which issue(s) this PR closes:
+
+Closes #
```

Comment:
```suggestion
- Closes #
```

By adding the hyphen and a space (`- `) GitHub will show the title of the issue.

Reply:
Added. Thanks! Resolving.